docker/docker-compose.yml

```diff
@@ -23,7 +23,7 @@ services:
       /usr/local/bin/integritee-service --clean-reset --ws-external -M integritee-worker-1 -T wss://integritee-worker-1 
       -u ws://integritee-node -U ws://integritee-worker-1 -P 2011 -w 2101 -p 9912 -h 4645
       run --dev --skip-ra"
-    restart: always
+    restart: "no"
```

Comment:
Just a minor change, discovered while running the tests in docker. It's confusing in a test setting when the working automatically restarts after a crash.